SimplyWealthProject/SimplyWealthApp/views.py

```diff
@@ -27,7 +101,16 @@ def get_ticker_details(request):
         stock_name = form.get('stockName').strip().upper()
         ticker_details_endpoint_url = f"https://api.polygon.io/v3/reference/tickers/{stock_name}?apiKey=LnR21zv6euM7KmY_HafxN9XgwdnmltXE"
         response = requests.get(ticker_details_endpoint_url).json()
-        output_response = {'results':response['results']}
+        
+        output_response = {'results':json.dumps(response['results'])}
+        user_profile = UserProfile.objects.get(user=request.user)
+        user_stock_portfolio = UserStockPortfolio.objects.filter(user = user_profile, stock_symbol=stock_name)
+        for record in user_stock_portfolio:
+            if record.stock_units > 0:
+                output_response['sell_option'] = True
+            else:
+                output_response['sell_option'] = False
+            
```

Comment:
Line117 is wrong. you want to change 
        ticker_timeseries_endpoint_url = f"https://api.polygon.io/v2/aggs/ticker/AAPL/range/1/day/{thirty_days_ago}/{curr_date}?adjusted=true&sort=desc&limit=30&apiKey=LnR21zv6euM7KmY_HafxN9XgwdnmltXE"


AAPL to {stockname}